src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -158,162 +208,236 @@ const PointShare = props => {
         />
       )}
       <Header
-        title="SHARE POINTS"
+        title="SCRIBBLE STADIUM"
         displayMenu={true}
         pointsRemaining={true}
         points={totalPoints}
         teamName={true}
       />
-      <QuestionCircleOutlined
-        className="question-icon"
-        onClick={() => {
-          setModalVisible(true);
-        }}
-      />
-      <InstructionsModal
-        modalVisible={modalVisible}
-        handleCancel={() => {
-          setModalVisible(false);
-        }}
-        handleOk={() => {
-          setModalVisible(false);
-        }}
-        instructions={modalInstructions.sharePoints}
-      />
-      <div className="point-share-container">
-        <Row className="team-row">
-          <Col>
-            <Row className="teammate-one">
-              <img
-                className="teammate-one-avatar"
-                src={props.team.child1.AvatarURL}
-                alt="Child Avatar"
-              />
-            </Row>
-            <Row className="teammate-two">
-              <img
-                className="teammate-one-avatar"
-                src={props.team.child2.AvatarURL}
-                alt="Child Avatar"
+
+      <div className="point-share-rectangle">
+        <div className="point-share-text">
+          <h2 className="point-share-text-heading">POINTS SHARING</h2>
+          <p className="point-share-paragraph">{`Now it is time to split up 100 points across your squad's portfolio of stories and drawings.  You must put a minimum of 10 points on each and the total among all 4 must add up to 100 points.`}</p>
+
+          <p className="point-share-paragraph">
+            {`As you read both stories and look at both drawings, think about which ones best reflect the characters,  plot, and settings from the chapters you read.  Share the most points with the ones you think are the best.`}{' '}
+          </p>
+        </div>
+      </div>
+
+      <div className="point-share-boxes-container">
+        <div className="point-share-box child-one">
+          <div className="point-share-child-info">
+            <img src={childOneImg} alt="child-one-avatar" />
+            <p className="point-share-heading">{`${user.characterName.toUpperCase()} (YOU)`}</p>
```

Comment:
.toUpperCase() was required here to ensure the rendering in all caps, no matter how the characterName data comes in.